src/pint/models/timing_model.py

```diff
@@ -366,6 +366,12 @@ def validate(self):
             self.T2CMETHOD.value = "IAU2000B"
         if self.UNITS.value not in [None, "TDB"]:
             raise ValueError("PINT only supports 'UNITS TDB'")
+        if not self.START.frozen:
```

Comment:
I think this sort of thing would be better put in the mjdParameter when it was created? This would avoid special-casing anything here.